svm/src/transaction_processor.rs

```diff
@@ -763,7 +744,7 @@ impl<FG: ForkGraph> TransactionBatchProcessor<FG> {
         }
     }
 
-    fn program_modification_slot<CB: TransactionProcessingCallback>(
+    pub fn program_modification_slot<CB: TransactionProcessingCallback>(
```

Comment:
Since this is now part of the public interface, adding a doc comment would be useful.